arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,104 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/powerOff": {
```

Comment:
Is there a difference between powerOff and stop (in terms of billing?)? It might be better if we make the nomenclature consistent - powerOff/powerOn or start/stop. You also have a forceReboot.